test/features/install_jobs_service.feature

```diff
@@ -108,3 +108,41 @@ Feature: Install Kogito Jobs Service
       """
 
     Then GraphQL request on Data Index service returns Jobs ID "jobs-service-data-index-id" within 2 minutes
+
+#####
+
+  # Disabled until KOGITO-2678 is done
+  @disabled 
+  @externalcomponent
+  @keycloak
+  @security
+  Scenario: Install Kogito Jobs Service with Keycloak security
+    Given Kogito Operator is deployed with Keycloak operator
+    And Keycloak instance is deployed
+    And Keycloak realm "kogito-realm" is deployed
+    And Keycloak client "kogito-jobs-service" is deployed
+    And Keycloak user "my-user" with password "my-password" is deployed
+    And Stores Keycloak endpoint into variable "auth-server-url"
+
+    When Install Kogito Jobs Service with 1 replicas with configuration:
+      | runtime-env  | quarkus.oidc.tenant-enabled                          | true                                         |
+      | runtime-env  | quarkus.oidc.tls.verification                        | none                                         |
+      | runtime-env  | quarkus.oidc.auth-server-url                         | {auth-server-url}/auth/realms/kogito-realm   |
```

Comment:
Keycloak operator creates a single pod without deployment and it exposes via a service, so we can't access on it via the deployment internal hostname.

Reply:
From where it comes the `auth-server-url`? I'd say to always use the internal service hostname instead, but I'm not sure if I understood your comment/approach :sweat_smile:

Reply:
I made some enhancements in this scenario and hopefully it will be easier to follow now. There is no longer an "auth-server-url" param :D 
Thanks for your comment! And let me know if it's ok now.